boards/BMS/Inc/Io/Io_SharedFreqOnlyPwmInputConfig.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+#define MAX_NUM_OF_FREQ_ONLY_PWM_INPUTS 2
```

Comment:
Let's leave this file in but with only `pragma once`, then we will define a default `MAX_NUM_OF_FREQ_ONLY_PWM_INPUTS ` there

Reply:
Fixed

Reply:
I don't see the changes. To be clear, I'm saying we should remove `#define MAX_NUM_OF_FREQ_ONLY_PWM_INPUTS 2` except for `boards/FSM/Inc/Io/Io_SharedFreqOnlyPwmInputConfig.h`

Reply:
Oh whoops I misinterpreted and thought you wanted ``#pragma once` removed